index.html

```diff
@@ -67,31 +67,32 @@ <h1 class="header__logo">devfinder</h1>
           class="profile__image"
         />
         <div class="profile__text">
-          <div class="profile__name-date">
-            <h2 class="profile__name">The Octocat</h2>
-            <span class="profile__date">Joined 25 Jan 2011</span>
+          <div class="profile__infos">
+            <img
+              src="./assets/Bitmap.png"
+              alt="Imagem de perfil"
+              class="profile__image--responsive"
+            />
+            <div class="profile__name-date-link">
+              <h2 class="profile__name">The Octocat</h2>
+              <span class="profile__date">Joined 25 Jan 2011</span>
+              <h3 class="profile__link">@octocat</h3>
+            </div>
           </div>
-          <h3 class="profile__link">@octocat</h3>
           <p class="profile__description">
             Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Donec
             odio. Quisque volutpat mattis eros.
           </p>
           <div class="statistics">
-            <div class="statistics__repo-number">
-              <h4 class="statistics__title">Repos</h4>
-              <span class="statistics__number">8</span>
-            </div>
-            <div class="statistics__followers-number">
-              <h4 class="statistics__title">Followers</h4>
-              <span class="statistics__number">3938</span>
-            </div>
-            <div class="statistics__following-number">
-              <h4 class="statistics__title">Following</h4>
-              <span class="statistics__number">9</span>
-            </div>
+            <h4 class="statistics__title">Repos</h4>
+            <h4 class="statistics__title">Followers</h4>
+            <h4 class="statistics__title">Following</h4>
+            <span class="statistics__number">8</span>
+            <span class="statistics__number">3938</span>
+            <span class="statistics__number">9</span>
           </div>
           <div class="social-media">
-            <div>
+            <div class="social-media__local">

```

Comment:
Com o `social-nedia__box` criado, tu poderia usar em todos os box de social media com base, e o `local` como modificador. Que achas?

```suggestion
            <div class="social-media__box social-nedia__box--local">
```